common/shlibs

```diff
@@ -2404,6 +2404,7 @@ libpinktrace_0.9.so.9 pinktrace-0.9.1_1
 libsyd_1.0.so.0 sydbox-1.0.2_1
 libcsync.so.0 csync-0.50.0_1
 libcalligrasheetsodf.so.17 calligra-sheets-3.2.0_1
+libkokross.so.17 calligra-libs-3.2.0_3
```

Comment:
```suggestion
libkokross.so.17 calligra-libs-3.2.0_1
```
Just go with revision 1, there

Reply:
No, `libkokross.so.17` isn't available in `_2`, I think some dependencies changed and it's available now.

Reply:
So it is available in 3.2.1_3 ? Not .0_ ?

Reply:
Ah, yes, it's available in `3.2.1_3` not in `3.2.1_2` and `3.2.0`